### DISCUSSION TITLE
[offtopic?] OTGW - Outside Temperature correction

### DISCUSSION THREAD

**Discussion by DaveDavenport** [rvdbreemen/OTGW-firmware#9] [General]
Not directly OTGW-firmware related, but I wonder how other people work with this.

I installed OTGW a while back, one of the ideas was to use OTGW to provide outside temperature to my thermostat (a remeha iSense v19). In the hope to get a better control (by having more relevant info).

I recently (around 1 week ago) got this setup and now an outside temperature is provided to my thermostat (thanks to OTGW).

However this is where I ran into problems, I set my thermostat control to use outside temperature with inside sensor correction, it just heats my room to 17.5 degrees day and night, and that is it. It feels like it is completely ignoring my setpoint.

Now I guess I can improve this by tweaking the heating curve, but I cannot find a good strategy to get this going?
So my question is:

1.) is it worth it? or does it not help that much.
2.) Is there are common practice to setting up the heating curve?

If these discussions are desired here, please let me know, I will remove it.

R,

D

**Comment by rvdbreemen** [comment]
@DaveDavenport I have no experience with setting the heatcurve, or sending back the OT to thru the gateway. So I could not give you any comment that makes sense.

The [Domotica forum - has a specific section for the OpenTherm Gateway](https://www.domoticaforum.eu/viewforum.php?f=75&sid=f589c35a86db563a58dcd0d23e4b4dc1) for these discussions. You could be in better luck there.
